src/app/components/articles/article/body/body.component.scss

```diff
@@ -0,0 +1,29 @@
+@use "/src/styles.scss" as *;
+
+.body {
+  background-color: $main-background-color;
+  color: $main-text-color;
+  padding-left: $big-border; 
+  padding-right: $big-border;
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+}
+
+  h2 {
```

Comment:
wrap all those tag targets inside the .body class

Reply:
Done.